test/vyasa/sangh_test.exs

```diff
@@ -27,9 +27,6 @@ defmodule Vyasa.SanghTest do
       assert session.id == "7488a646-e31f-11e4-aace-600308960662"
     end
 
-    test "create_session/1 with invalid data returns error changeset" do
-      assert {:error, %Ecto.Changeset{}} = Sangh.create_session(@invalid_attrs)
-    end
 
```

Comment:
some other time, I think it would be good if you could run me thru some elxir/liveview test-writing best practices @ks0m1c